example/aws_s3_presigned_url.dart

```diff
@@ -0,0 +1,53 @@
+import 'dart:io';
+import 'package:aws_auth/aws_auth.dart';
+import 'package:http/http.dart' as http;
+
+const AWS_KEY = 'YOUR_AWS_KEY';
+const AWS_SECRET = 'YOUR_AWS_SECRET';
+const AWS_REGION = 'region';
+const BUCKET_NAME = 'BUCKET';
+
+AWSRequest createPresignS3Request(
+    AWS4Signer signer, Duration expires, String bucketName, String bucketPath,
+    {bool public = false}) {
+  // create request
+  final req = AWSRequest(
+      //'https://$bucketName.${signer.region}.digitaloceanspaces.com$bucketPath',
+      'https://$bucketName.s3.${signer.region}.amazonaws.com$bucketPath',
```

Comment:
Did you check how it works in case of paths with special characters? (e.g. spaces)